Packages/com.unity.inputsystem/InputSystem/NativeInputRuntime.cs

```diff
@@ -107,6 +107,19 @@ public Action onShutdown
             }
         }
 
+        public Action<bool> onFocusChanged
+        {
+            set
+            {
+                if (value == null)
+                    Application.focusChanged -= OnFocusChanged;
```

Comment:
We can't merge things that break the build for users that don't have the respective changes. This needs to either use #if based on Unity version where the native change has landed or use reflection to hook into the functionality when available.